docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -127,64 +127,95 @@ This example leverages the Sui TypeScript SDK to subscribe to events the package
+
+    const executeMoveCall = (method: "small" | "large") => {
+        const txb = new TransactionBlock();
+
+        if (method === "small") {
+        txb.moveCall({
+            arguments: [txb.pure.u64(0)],
+            target: `${SICBOGAME_PACKAGE_ID}::sicbogame::create_game`,
+        });
+        } else {
+        txb.moveCall({
+            arguments: [txb.pure.u64(1)],
+            target: `${SICBOGAME_PACKAGE_ID}::sicbogame::create_game`,
+        });
+        }
+
+        signAndExecute(
+        {
+            transactionBlock: txb,
+            options: {
+            showEffects: true,
+            showObjectChanges: true,
+            },
+        },
+        {
+            onSuccess: (tx) => {
+                client.
+                    queryEvents({
+                        query:{Transaction:tx.digest}
+                    }).then(PaginatedEvents=> {
+                        const events = JSON.stringify(PaginatedEvents.data[0].parsedJson);
+                    
+                        if (events.length > 0) {
+                          onCreated(events)
+                        } else {
+                          onCreated('No events found for the given criteria.');
+                        }
+                      })
+                      .catch(error => { 
+                        console.error('Error querying events:', error);
+                      });
+            },
+        },
+        );
+    };
+    return (
+        <>
+        <Heading size="3">Game Start</Heading>
+
+        <Flex direction="column" gap="2">
+            <Flex direction="row" gap="2">
+            <Button onClick={() => executeMoveCall("small")}>
+                small
+            </Button>
+            <Button onClick={() => executeMoveCall("large")}>
+                large
+            </Button>
+            </Flex>
+        </Flex>
+        </>
+    );
+}
 ```
 
 ### Response
 
-When the subscribed to event fires, the example displays the following JSON representation of the event.
+Example transaction：https://suiexplorer.com/txblock/2hg7QQvNh7DnmmMaanAZ7hRYCAUimPJTN3bJRnpo29ej?network=testnet
 
-```json
-subscribeEvent {
-  "id": {
-    "txDigest": "HkCBeBLQbpKBYXmuQeTM98zprUqaACRkjKmmtvC6MiP1",
-    "eventSeq": "0"
-  },
-  "packageId": "0x2d6733a32e957430324196dc5d786d7c839f3c7bbfd92b83c469448b988413b1",
-  "transactionModule": "coin_flip",
-  "sender": "0x46f184f2d68007e4344fffe603c4ccacd22f4f28c47f321826e83619dede558e",
-  "type": "0x2d6733a32e957430324196dc5d786d7c839f3c7bbfd92b83c469448b988413b1::coin_flip::Outcome",
-  "parsedJson": {
-    "bet_amount": "4000000000",
-    "game_id": "0xa7e1fb3c18a88d048b75532de219645410705fa48bfb8b13e8dbdbb7f4b9bbce",
-    "guess": 0,
-    "player_won": true
-  },
-  "bcs": "3oWWjWKRVu115bnnZphyDcJ8EyF9X4pgVguwhEtcsVpBf74B6RywQupm2X",
-  "timestampMs": "1687912116638"
-}
-```
+I can get the value of parsedJson in the example below, and its final value is {"msg":"You Win :)"} .
```

Comment:
```suggestion
As the returned JSON shows, the value of `parsedJson` is `{"msg":"You Win :)"}` .
```

Reply:
Thanks @ronny-mysten ，you're probably right.